common/compositor/va/varenderer.cpp

```diff
@@ -487,18 +490,25 @@ bool VARenderer::UpdateCaps() {
   return true;
 }
 
-uint32_t VARenderer::HWCTransformToVA(uint32_t transform) {
-    switch (transform) {
-        case kTransform270:
-            return VA_ROTATION_270;
-        case kTransform180:
-            return VA_ROTATION_180;
-        case kTransform90:
-            return VA_ROTATION_90;
-        default:
-            break;
-    }
-    return VA_ROTATION_NONE;
+uint32_t VARenderer::HWCRotationToVA(uint32_t transform) {
+  if(transform & kTransform270)
+    return VA_ROTATION_90;
+  if(transform & kTransform180)
+    return VA_ROTATION_180;
+  if(transform & kTransform90)
+    return VA_ROTATION_270;
```

Comment:
hmm why do  u change this to 270 ? Is this something u forgot ? why change original code ?